smalltalksrc/VMMakerTests/VMJittedPrimitivesTest.class.st

```diff
@@ -97,25 +97,6 @@ VMJittedPrimitivesTest >> new64BitIndexableOfSize: aSize [
 	^ self newBitIndexableOfSize: aSize bytesPerSlot: 8 format: memory sixtyFourBitIndexableFormat
 ]
 
-{ #category : #helpers }
-VMJittedPrimitivesTest >> new8BitIndexableOfSize: aSize [
-
-	^ self newBitIndexableOfSize: aSize bytesPerSlot: 1 format: memory firstByteFormat
-]
-
-{ #category : #helpers }
-VMJittedPrimitivesTest >> newBitIndexableOfSize: aSize bytesPerSlot: bytesPerSlot format: format [
-
-	| padding numberOfWordSizeSlots desiredByteSize |
-	desiredByteSize := aSize * bytesPerSlot roundUpTo: self wordSize.
-	numberOfWordSizeSlots := desiredByteSize / self wordSize.
-	padding := desiredByteSize / bytesPerSlot - aSize.
-	^ memory
-		  allocateNewSpaceSlots: numberOfWordSizeSlots
-		  format: format + padding
-		  classIndex: self nextOrdinaryClassIndex
-]
-
```

Comment:
Ok, IIRC, these methods are duplicated in the superclass so removing them here is safe.